cmd/kbld/kbld.go

```diff
@@ -18,7 +18,7 @@ import (
 func main() {
 	rand.Seed(time.Now().UTC().UnixNano())
 
-	log.SetOutput(ioutil.Discard)
+	log.SetOutput(io.Discard)
```

Comment:
Here also should we use the `os` package only as it is already imported so we can ignore importing `io`?